scout/adapter/mongo/hgnc.py

```diff
@@ -131,34 +131,19 @@ def hgnc_genes(self, hgnc_symbol, build="37", search=False):
             )
         return self.hgnc_collection.find({"build": build, "aliases": hgnc_symbol})
 
-    def hgnc_genes_count(self, hgnc_symbol, build="37", search=False):
-        """Count all hgnc genes that match a hgnc symbol. Replaces depricated
+    def hgnc_genes_find_one(self, hgnc_symbol):
```

Comment:
Hi, I meant that this function is not necessary. You can call the store.hgnc_collection.find_one directly from the functions that now call this one.

Reply:
I think we should have an interface layer towards MongoDB. This makes the code more likely to be maintainable next time the Pymongo library is changed.

Reply:
I agree with @mikaell . This is the reason why we got stuck on an old version of pymongo in the first place, it's because we used pymongo methods directly instead of defining an api

Reply:
I agree as well on the principle, but in this case it's a function of one line that calls a direct pymongo method that is called only 2 times in the whole repo. But if you want to keep it this way it's fine by me!